charts/newrelic-prometheus-agent/values.yaml

```diff
@@ -440,3 +440,6 @@ config:
   # This configuration should be used as a workaround whenever kubernetes and static job do not cover a particular use-case.
   # @default -- `[]`
   extra_scrape_configs: []
+
+  metric_type_override:
```

Comment:
could you add some comments to this field?

Reply:
added